client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -364,6 +396,11 @@ export default {
 
   data () {
     return {
+      activeId: 'recTags',
+      addonProps: {
+        segmentId: this.segment.id,
```

Comment:
I think this should fetched via store. We should design it like so, that we change the core as less as possible when writing an addon

Reply:
But isn't it like we need the id to fetch a single segment from the store? Currently the addon requires the segment.id from its corresponding parent to set the mutation in the store. I'm not quite sure how we can design this otherwise.